src/content/en/updates/2017/06/budget-api.md

```diff
@@ -0,0 +1,189 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: The Budget API allows developers to perform background actions without notifying users, enabling use cases like silent push.
+
+{# wf_updated_on: 2017-06-07 #}
+{# wf_published_on: 2017-06-07 #}
+{# wf_tags: javascript,push,budget,api #}
+{# wf_featured_image: /web/updates/images/generic/info.png #}
+
+# Introduction to the Budget API {: .page-title }
+
+The Push Messaging API enables us to send notifications to a user even when the
+browser is closed. Many developers want to be able to use this messaging to
+update and synchornise content without the browser being open, but the API has
+one important restriction: you must always display a notification for every
+single push message recieved.
+
+Being able to send a push message to synchronize data on a user's device can be
+extremely useful for users and developers, but allowing a web app to do work in
+the background without the user knowing is open to abuse.
+
+The [Budget
+](https://wicg.github.io/budget-api/)[API](https://wicg.github.io/budget-api/),
+is a new API designed to allow developers to perform background work without
+notifying the user, such as silent push or performing a background fetch. In
+Chrome version 60 and above you'll be able to start using this API and the
+Chrome team is eager to get feedback from developers.
+
+To allow developers to consume a user's resources in the background, the web
```

Comment:
Not required, but since this is the subject of the post, it could be closer to the top.